bazel/get_workspace_status

```diff
@@ -42,5 +42,5 @@ echo "BUILD_SCM_STATUS ${tree_status}"
 echo "STABLE_BUILD_SCM_STATUS ${tree_status}"
 
 # Generate a build version number to label built binaries in the format specified in evergreen/archive-evergreen-rhel7-amd64.sh
-git_version_number=$(git describe --tags --dirty --always --match 'v[0-9]*' | cut -c 2-)
+git_version_number=$(git describe --tags --dirty --always --match 'v[0-9]*' --abbrev=7 | cut -c 2-)
```

Comment:
Per git describe [docs](https://git-scm.com/docs/git-describe#_options), we can add the `--abbrev` flag here so that this command always returns the abbreviated commit id with a length of 7 (instead of a variable amount).